### PR TITLE
Documentation: Updated Example Configuration for Trusted Clusters.

### PR DESCRIPTION
**Pull request by russjones**
**Purpose**

Documentation for `TrustedCluster` resource introduced in https://github.com/gravitational/teleport/pull/815.

**Implementation**

* Updated `Example Configuration` section.'
* Updated OpenSSH compatibility documentation to show how you can use the `proxy` subsystem to connect to nodes within a Trusted Cluster.
* Minor update to `docker/Makefile` to allow entry into `two-node` as well.

### CODE REVIEW

**Comment by russjones** [comment]
@gravitational-jenkins test this please

File: docs/admin-guide.md [hunk 1]
```diff
@@ -523,72 +523,71 @@ This setup works as follows:
...
-other. To verify, run this on "cluster-b":
+To add behind-the-firewall machines and restrict access so only users with role
+"admin" can access them, do the following:
+
+1. Create a static or dynamic token on `main` that will be used by `cluster-b`
+   to join `main`. A dynamic token can be generated by running:
+   `tctl nodes add --ttl=5m --roles=trustedcluster` and a static token can be
+   generated out-of-band and added to your configuration file like so:
+
+        auth_service:
```
Comment (klizhentas) [inline docs/admin-guide.md]: can you use markdown syntax for yaml files? 

```yaml
auth_service:

```

File: docs/admin-guide.md [hunk 2]
```diff
@@ -523,72 +523,71 @@ This setup works as follows:
...
+            # using a tool like `pwgen` to generate sufficiently random
+            # tokens of length greater than 32 bytes
+            - "trustedcluster:fake-token"
+
+1. Create a `TrustedCluster` resource on `cluster-b` that tells `cluster-b`
+   how to connect to main and the token created in the previous step for
+   authorization and authentication. To do that, copy the resource below
+   to a file called `trusted-cluster.yaml`.
+
+        kind: trusted_cluster
```
Comment (klizhentas) [inline docs/admin-guide.md]: same here

File: docs/admin-guide.md [hunk 3]
```diff
@@ -523,72 +523,71 @@ This setup works as follows:
...
+          tunnel_addr: <main-addr>:3024
+          web_proxy_addr: <main-addr>:3080
+
+1. Notice how we defined `roles` in the `TrustedCluster` resource. This is
+   the role assumed by any user when they connect from `main` to `cluster-b`.
+   That means we need to make sure the `admin` role exists in Teleport and we
+   need it associate it with a user (let's say the user is named "john"). To do
+   that, copy the resource below to a file called`admin-role.yaml`.
+
+        kind: role
```
Comment (klizhentas) [inline docs/admin-guide.md]: same here

File: docs/admin-guide.md [hunk 4]
```diff
@@ -683,6 +682,13 @@ Host *.work.example.com
     HostName %h
     Port 3022
     ProxyCommand ssh -p 3023 %r@work.example.com -s proxy:%h:%p
+
+# when connecting to a node within a trusted cluster with name "remote-cluster",
+# add the name of the cluster to the invocation of the proxy subsystem.
+Host *.remote-cluster.example.com
```
Comment (klizhentas) [inline docs/admin-guide.md]: use verbatim blocks (3 backticks` here as well

**Review by klizhentas — COMMENTED** [review]
looks good except formatting

File: docs/admin-guide.md (hunk 4 above)
Comment (russjones) [inline docs/admin-guide.md]: It is in a verbatim block?

**Review by klizhentas — APPROVED** [review]

**Pull request merged**